resources/dicts/events/ceremonies/ceremony_text.json

```diff
@@ -800,38 +800,91 @@
             "bold": [
                 "(prefix)paw may have butted heads against nearly every Clan member they have, but being able to argue was a blessing in a medicine cat, as m_c will have to continue shoving odd mixtures down everyone's throat. StarClan will be watching them with pride."
             ],
-            "calm": [],
-            "careful": [],
-            "charismatic": [],
-            "childish": [],
-            "cold": [],
-            "compassionate": [],
-            "confident": [],
-            "daring": [],
-            "empathetic": [],
+            "calm": [
+                "(prefix)paw has proven themsleves with one of the most invaluble traits a medicine cat can have, a level head under pressure. The stars of Silverpelt twinkle in joy and acceptance as m_c recives their full name and duites as a full medicine cat."
```

Comment:
I would use a hyphen here instead of a comma
"...a medicine cat can have - a level head under pressure."
also duites

Reply:
i also think a semi-colon would be suitable, "... a medicine cat can have: a level head under pressure."